docs/javascript/v2/guides/javascript-quickstart.mdx

```diff
@@ -3,33 +3,51 @@ title: JavaScript Quickstart Guide
 nav: 2.1
 ---
 
-## Getting started
+## Overview
 
-Hello there! In this guide, we'll build a video conferencing application using the 100ms JavaScript SDK. This guide doesn't make any assumptions related to
-the framework or library choices. In case you plan to use React, do check our React quickstart as well.
+Hello there! In this guide, we'll walk you through step-by-step instructions to create a  video conferencing application using the 100ms JavaScript SDK and test it locally on your device. In case you plan to use React, do check our [React quickstart](https://www.100ms.live/docs/javascript/v2/guides/react-quickstart) as well.
 
 TL;DR - You can find the link to the complete example over [here](https://codesandbox.io/s/hms-js-quickstart-5rmes).
 
-### Prerequisites
+## Setup a local Javascript app
+In this walkthrough, we’ll use ParcelJS a web application bundler that requires zero configuration. If you wish to use any other library or framework, you can set up your app using that. 
 
-To get started you must be familiar with rudimentary JavaScript and have [nodejs](https://nodejs.org/) installed on your machine.
+### Initialize a project
+You can initialize a project using `npm init` or `yarn init`. While initializing the project you can enter the details for your project as you wish.
 
-### Installing the dependencies
+### Install Parcel
+Then, install Parcel into your app using Yarn `yarn add --dev parcel` or when using npm run `npm install --save-dev parcel`.
 
-```bash
-npm install --save @100mslive/hms-video-store@latest
+### Project setup
+Now that parcel is installed, let’s create basic source files for our video-conferencing application. Let’s create the files and folders in the below manner:
```

Comment:
I noticed that we're using "parcel" in lower-case here. this is a common problem in our docs - there are lots of places where we use "react" instead of "React", "webrtc" instead of WebRTC. can these be caught by the linting work that we're doing?